packages/salesforcedx-utils-vscode/src/test/testrunner.ts

```diff
@@ -34,6 +34,23 @@ function configure(mochaOpts: any): void {
     // default to 'mocha-multi-reporters' (to get xunit.xml result)
     mochaOpts.reporter = 'mocha-multi-reporters';
   }
+  if (!mochaOpts.reporterOptions) {
+    let xmlPath = '';
+    if (process.platform === 'win32') {
+      xmlPath = paths.normalize(paths.join(process.cwd(), '..', '..'));
+    }
+    mochaOpts.reporterOptions = {
+      reporterEnabled: 'mocha-junit-reporter, xunit, spec',
+      mochaJunitReporterReporterOptions: {
+        mochaFile: xmlPath
+          ? paths.join(xmlPath, 'junit-custom.xml')
+          : 'junit-custom.xml'
+      },
+      xunitReporterOptions: {
+        output: xmlPath ? paths.join(xmlPath, 'xunit.xml') : 'xunit.xml'
+      }
+    };
+  }
```

Comment:
Sets up the reporter options. This is essentially what the config.json files do for the non-programmatic runs of mocha. There were some oddities on Windows where the mocha execution would be inside the downloaded version of vscode which is why you see the if case.

Reply:
Would be good to add that as a comment.